### PR TITLE
Bz1990481-R: Add commands for changing cronjob CR of Compliance Operator

### PR DESCRIPTION
**Pull request by dfitzmau**
BZ1990481:Add commands for chaning cronjob CR of Compliance Operator

Version(s):
4.8+

Issue:
[BZ1990481](https://bugzilla.redhat.com/show_bug.cgi?id=1990481)

Link to docs preview:
[ComplianceSuite custom resource lifecycle and debugging](http://file.emea.redhat.com/dfitzmau/01-25-23/BZ1990481-R/security/compliance_operator/compliance-operator-troubleshooting.html#compliance-suite-lifecycle-debugging_compliance-anatomy-sections)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by dfitzmau** [comment]
Note: Appended `-R` to branch name, as the PR replaces another PR where the incorrect upstream branch was targeted (4.8).

File: modules/compliance-anatomy.adoc [hunk 1]
```diff
@@ -109,6 +109,40 @@ NAME                                           SCHEDULE    SUSPEND   ACTIVE   LA
...
 For the most important issues, events are emitted. View them with `oc describe compliancesuites/<name>`. The `Suite` objects also have a `Status` subresource that is updated when any of `Scan` objects that belong to this suite update their `Status` subresource. After all expected scans are created, control is passed to the scan controller.
 
+Sometimes, the `ComplianceSuite` CR cannot create a scan if the scan exceeds the default memory limit of the Compliance Operator. Exceeding this limit can result in an Out of Memory (OOM) error message. You can resolve this issue by changing the configuration of the `CronJob` CR, so that you can increase the memory limit for a scan. The following sequence of actions demonstrates an example of how you can access a edit a `suitererunner` resource:
+
+[source,terminal,subs="+quotes"]
+----
+$ oc get cronjob -n openshift-compliance
+----
+
+Command returns a list of `suitererunner` resources that the Compliance Operator emits in a `CronJob` CR.
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply modules/compliance-anatomy.adoc]: Thanks for clarifying. Updated.

File: modules/compliance-anatomy.adoc [hunk 2]
```diff
@@ -109,6 +110,28 @@ NAME                                           SCHEDULE    SUSPEND   ACTIVE   LA
 
 For the most important issues, events are emitted. View them with `oc describe compliancesuites/<name>`. The `Suite` objects also have a `Status` subresource that is updated when any of `Scan` objects that belong to this suite update their `Status` subresource. After all expected scans are created, control is passed to the scan controller.
 
+Sometimes, the `ComplianceSuite` CR cannot create a scan if the scan exceeds the default memory limit of the Compliance Operator. Exceeding this limit can result in an Out of Memory (OOM) error message. You can increase the memory for the Compliance Operator by setting custom resource limits.
```
Comment (rhmdnd) [inline modules/compliance-anatomy.adoc:113]: It looks like we still have a similar section at the bottom of this page:

http://file.emea.redhat.com/dfitzmau/01-25-23/BZ1990481-R/security/compliance_operator/compliance-operator-troubleshooting.html#operator-resource-constraints_compliance-anatomy-sections

Do we want to keep that separate from this? Or do we want to take the opportunity to consolidate that section too?

**Comment by dfitzmau** [comment]
Closing PR based on https://bugzilla.redhat.com/show_bug.cgi?id=1990481

**Pull request closed without merging**